jobs/signing/SignVsix.proj

```diff
@@ -10,8 +10,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <FilesToSign Include="$(OutDir)\**\*.vsix">
-      <Authenticode>VsixSHA2</Authenticode>
+    <FilesToSign Include="$(OutDir)\vsix\extension.signature.p7s">
```

Comment:
Hmm I also don't understand this change

Reply:
This is the update for signing. We can't sign the vsix with VsixSha2 anymore because it messes up the signing of the manifest.

Reply:
This is how the docs say we should sign